ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -685,6 +685,31 @@ public async Task EditMission(int missionNo)
             }
         }
 
+        [Command("edytuj-nazwe-misji")]
+        [Summary("Edycja nazwy już utworzonej misji.")]
+        [ContextDMOrChannel]
+        public async Task MissionName([Remainder] string newTitle)
+        {
+            var signups = _map.GetService<SignupsData>();
```

Comment:
Please use the `SignupsData` property instead of getting the service from provider.

Reply:
Done.